src/inference/include/openvino/runtime/auto/properties.hpp

```diff
@@ -28,5 +28,51 @@ static constexpr Property<bool> enable_startup_fallback{"ENABLE_STARTUP_FALLBACK
  * selected device
  */
 static constexpr Property<bool> enable_runtime_fallback{"ENABLE_RUNTIME_FALLBACK"};
+
+/**
+ * @brief Enum to define the policy of scheduling inference request to target device in cumulative throughput mode on
+ * AUTO
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+enum class SchedulePolicy {
+    ROUND_ROBIN = 0,            // will schedule the infer request using round robin policy
+    DEVICE_PRIORITY = 1,        // will schedule the infer request based on the device priority
+    DEFAULT = DEVICE_PRIORITY,  //!<  Default schedule policy is DEVICE_PRIORITY
+};
+
+/** @cond INTERNAL */
+inline std::ostream& operator<<(std::ostream& os, const SchedulePolicy& policy) {
+    switch (policy) {
+    case SchedulePolicy::ROUND_ROBIN:
+        return os << "ROUND_ROBIN";
+    case SchedulePolicy::DEVICE_PRIORITY:
+        return os << "DEVICE_PRIORITY";
+    default:
+        OPENVINO_THROW("Unsupported schedule policy value");
+    }
+}
+
+inline std::istream& operator>>(std::istream& is, SchedulePolicy& policy) {
+    std::string str;
+    is >> str;
+    if (str == "ROUND_ROBIN") {
+        policy = SchedulePolicy::ROUND_ROBIN;
+    } else if (str == "DEVICE_PRIORITY") {
+        policy = SchedulePolicy::DEVICE_PRIORITY;
+    } else if (str == "DEFAULT") {
+        policy = SchedulePolicy::DEFAULT;
+    } else {
+        OPENVINO_THROW("Unsupported schedule policy: ", str);
+    }
+    return is;
+}
+/** @endcond */
+
+/**
+ * @brief High-level OpenVINO model policy hint
+ * Defines what model should be provided with more performant bounded resource first
```

Comment:
Please update definition from model priority to schedule policy. 

"Defines what scheduling policy should be used in AUTO CUMULATIVE_THROUGHPUT or MULTI case "

Reply:
Updated